lib/l10n/intl_en.arb

```
@@ -767,6 +797,58 @@
+  "retry": "Retry",
+  
+  "@_comment_dispute_info": "Dispute Info Text",
+  "unknown": "Unknown",
+  "disputeWith": "Dispute with {role}: {counterparty}",
+  "@disputeWith": {
+    "placeholders": {
+      "role": { "type": "String" },
+      "counterparty": { "type": "String" }
+    }
+  },
+  "orderIdLabel": "Order ID",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)